gcloud-java-dns/src/test/java/com/google/gcloud/dns/DnsRecordTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gcloud.dns;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertNotEquals;
+
+import org.junit.Test;
+
+public class DnsRecordTest {
+
+  private static final String NAME = "example.com.";
+  private static final Integer TTL = 3600;
+  private static final DnsRecord.DnsRecordType TYPE = DnsRecord.DnsRecordType.AAAA;
+  private static final DnsRecord record = DnsRecord.builder(NAME, TYPE)
+          .ttl(TTL)
```

Comment:
wrong indent. see [this](https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent)